### ISSUE TITLE
Update to Vala conventions

### ISSUE THREAD

**Issue by desiderantes** [Serranya/sdl2-vala-lazy-foo#1]
This repo is awesome, truly awesome, however i see that you are using LazyFoo-s naming conventions. IMHO using Vala conventions would be better, since the audience for this kind of tutorials is composed mostly by Vala programmers.

**Comment by Serranya** [comment]
You are right. In some places I converted the names to the appropriate Vala names. But I missed quite a lot. Im putting this on my TODO list. Feel free to submit a pull request if you want to do some work by yourself.
